pkg/ocm/flag.go

```diff
@@ -21,12 +21,21 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"github.com/openshift/rosa/pkg/arguments"
 	"github.com/openshift/rosa/pkg/aws"
 	"github.com/openshift/rosa/pkg/logging"
 	rprtr "github.com/openshift/rosa/pkg/reporter"
 )
 
 var clusterKey string
+var mode string
+
+const (
+	ModeAuto   = "auto"
+	ModeManual = "manual"
+)
+
+var Modes = []string{ModeAuto, ModeManual}
```

Comment:
I'd say this is probably not a good candidate for an "ocm" flag, more like an "aws" flag, but that's me /nitpicking :)